sdk/iot/Azure.Iot.Hub.Service/src/DevicesClient.cs

```diff
@@ -99,7 +99,7 @@ public virtual Response<DeviceIdentity> GetIdentity(string deviceId, Cancellatio
         /// <summary>
         /// Delete a single device identity.
         /// </summary>
-        /// <param name="deviceIdentity">the device identity to delete. If no ETag is present on the device, then the condition must be equal to <see cref="IfMatchPrecondition.Unconditional"/> or equal to <see cref="IfMatchPrecondition.UnconditionalIfMatch"/>.</param>
+        /// <param name="deviceIdentity">the device identity to delete. If no ETag is present on the device, then the condition must be equal to <see cref="IfMatchPrecondition.UnconditionalIfMatch"/> or equal to <see cref="IfMatchPrecondition.UnconditionalIfMatch"/>.</param>
```

Comment:
This is now redundant, no need to say it equals "UnconditionalIfMatch" or "UnconditionalIfMatch"

Reply:
This hasn't been fixed yet